scrapers/JavLibrary_python.py

```diff
@@ -0,0 +1,586 @@
+        return html
+    jav_search_tree = lxml.html.fromstring(html.content)
+    jav_url = getxpath(xpath['url'], jav_search_tree)  # ./?v=javme5it6a
+    if jav_url:
+        jav_url = re.sub(r"^\.", "https://www.javlibrary.com/en", jav_url[0])
+        jav_main_html = sendRequest(jav_url, JAV_HEADERS)
+        return jav_main_html
+    else:
+        debug("[JAV] There is no result in search")
+        return None
+
+
+def buildlist_tagperf(data, type_scrape=""):
+    list_tmp = []
+    dict_jav = None
+    if type_scrape == "perf_jav":
+        dict_jav = data
+        data = data["performers"]
+    for i in range(0, len(data)):
+        y = data[i]
+        if y == "":
+            continue
+        if type_scrape == "perf_jav":
+            # Invert name
+            y = re.sub(r"([a-zA-Z]+)(\s)([a-zA-Z]+)", r"\3 \1", y)
+        if type_scrape == "tags" and y in IGNORE_TAGS:
+            continue
+        if type_scrape == "perf_jav" and dict_jav.get("performer_aliases"):
+            try:
+                list_tmp.append({"name": y, "aliases": dict_jav["performer_aliases"][i]})
+            except:
+                list_tmp.append({"name": y})
+        else:
+            list_tmp.append({"name": y})
+    # Adding personal fixed tags
+    if FIXED_TAGS and type_scrape == "tags":
+        list_tmp.append({"name": FIXED_TAGS})
+    return list_tmp
+
+
+def th_request_perfpage(page_url, perf_url):
+    # vl_star.php?s=afhvw
+    #debug("[DEBUG] Aliases Thread: {}".format(threading.get_ident()))
+    javlibrary_ja_html = sendRequest(page_url.replace("/en/", "/ja/"), JAV_HEADERS)
+    if javlibrary_ja_html:
+        javlibrary_perf_ja = lxml.html.fromstring(javlibrary_ja_html.content)
+        list_tmp = []
+        try:
+            for i in range(0, len(perf_url)):
+                list_tmp.append(javlibrary_perf_ja.xpath('//a[@href="' + perf_url[i] + '"]/text()')[0])
+            if list_tmp:
+                jav_result['performer_aliases'] = list_tmp
+                debug("[DEBUG] Got the aliases: {}".format(list_tmp))
+        except:
+            debug("[DEBUG] Error with the aliases")
+    else:
+        debug("[DEBUG] Can't get the Jap HTML")
+    return
+
+
+def th_imagetoBase64(imageurl, typevar):
+    #debug("[DEBUG] {} thread: {}".format(typevar,threading.get_ident()))
+    if type(imageurl) is list:
+        for image_index in range(0, len(imageurl)):
+            try:
+                base64image = base64.b64encode(requests.get(imageurl[image_index].replace("ps.jpg", "pl.jpg"), timeout=10).content)
+                imageurl[image_index] = "data:image/jpeg;base64," + base64image.decode('utf-8')
+            except:
+                debug("[DEBUG][{}] Failed to get the base64 of the image".format(typevar))
+        if typevar == "R18Series":
+            r18_result["series_image"] = imageurl
+    else:
+        try:
+            base64image = base64.b64encode(requests.get(imageurl.replace("ps.jpg", "pl.jpg"), timeout=10).content)
```

Comment:
i had an issue with the images
if the image cannot be retrieved (in this case a 403 error since headers were not sent) the content shouldnt be used 
the below seems to fix that, it should modified a bit and applied also to the `list` case above it
```python
            head = JAV_HEADERS
            if "r18.com/" in imageurl:
                head = R18_HEADERS
            img = requests.get(imageurl.replace("ps.jpg", "pl.jpg"), timeout=10, headers=head)
            if img.status_code != 200:
                debug(f"request error {img.status_code} for <{imageurl}>")
                return
            base64image = img.content
```